apache-bookkeeper/src/main/java/com/baeldung/BookkeeperLedgerExample.java

```diff
@@ -0,0 +1,72 @@
+package com.baeldung;
+
+
+import java.nio.ByteBuffer;
+import java.util.Iterator;
+import org.apache.bookkeeper.client.api.BookKeeper;
+import org.apache.bookkeeper.client.api.DigestType;
+import org.apache.bookkeeper.client.api.LedgerEntries;
+import org.apache.bookkeeper.client.api.LedgerEntry;
+import org.apache.bookkeeper.client.api.ReadHandle;
+import org.apache.bookkeeper.client.api.WriteHandle;
+import org.apache.bookkeeper.conf.ClientConfiguration;
+
+
+public class BookkeeperLedgerExample {
```

Comment:
I think this class can be renamed to just `BookkeeperLedger`, we don't typically include `Example` in the class name because majority of our code is examples